taskcluster/firefox_tv_taskgraph/target_tasks.py

```diff
@@ -7,11 +7,27 @@
 from taskgraph.target_tasks import _target_task, standard_filter
 
 
+def tag_filter(target_release_type, task, params):
+    release_type = task.attributes.get("release-type")
+    return standard_filter(task, params) and (release_type is None or release_type == target_release_type)
+
+
 @_target_task('default')
 def target_tasks_default(full_task_graph, parameters, graph_config):
     """Target the tasks which have indicated they should be run on this project
     via the `run_on_projects` attributes."""
-    def filter(task, params):
-        return standard_filter(task, params)
+
+    if parameters["tasks_for"] == "github-release":
+        release_type = parameters["release_type"]
+
+        if release_type == "LAT":
+            def filter(task, params):
+                return tag_filter("lat", task, params)
+        else:
+            def filter(task, params):
+                return tag_filter("production", task, params)
+    else:
+        def filter(task, params):
```

Comment:
nit: I don't love that this name shadows a built-in function name.

Reply:
That's a good point, though I'm following the pattern from `fenix`.
I'm not super familiar with `taskgraph`, but so far it seems like the existing convention

Reply:
I've followed this pattern from mozilla-central too. I've never taken the time to understand why it's okay to shadow this built-in.